### DISCUSSION TITLE
Unable to read Serial Data on uart

### DISCUSSION THREAD

**Discussion by iottrends** [ExpressLRS/ExpressLRS#1847] [Q&A]
Hi 
I have a DIY ESP8285-SX1280-2400 RX  module.
I am able to flash the hardware using the ExLRS configurator UART Method. firmware version 2.00
Flashing  succeed;
 LEDs are blinking and wifi hot ExpressLRS RX is available, 
I am able to connect to wifi and successfully open the webpage 10.0.0.1.
However **I want to read the serial output.** That is giving garbage output.
For Serial reading : 
1. I used the ExpressLRS Configurator Serial Monitor @420000 buad rate. This gives me Garbage output.
2. Then I tried connecting to the Arduino serial Monitor 115200 , 57600,9600 allof them giving the garbage output.
3. attached screenshot for reference.  
4.
![image](https://user-images.githubusercontent.com/80441545/190843793-0326307b-f1d3-412c-b974-b4d19dd0cef6.png)

Expresslrs serial port
![image](https://user-images.githubusercontent.com/80441545/190843829-6771f66f-8a52-4e8b-9f74-7242a1180c08.png)

![image](https://user-images.githubusercontent.com/80441545/190843849-51eab598-d12b-4bb1-9200-515ff4667c2f.png)

**Comment by Ncerzzk** [comment]
What data do you want to get through uart?
the "garbage output" is not "garbage", it may be crsf data, it's not string, so you could not read it directly.
if you just want to get debug log, uncomment `DEBUG_CRSF_NO_OUTPUT` and `DEBUG_LOG_VERBOSE` when building.(see `User Defines` in wiki)

the default baudrate is 420000, so 115200 or other baudrate is uncorrect.
